src/libLookup/Lookup.cpp

```diff
@@ -2706,3 +2831,13 @@ bool Lookup::GetIsServer() {
 
   return m_isServer;
 }
+
+bool Lookup::VerifyLookupNode(const VectorOfLookupNode& vecLookupNodes,
```

Comment:
still, putting a const reference seems not the solution to the thread-safety issue. i remember that `m_lookupNode` will have write operations somewhere. Can you take a look again?